projects/storefrontlib/shared/components/chat-messaging/avatar/avatar.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, Input } from '@angular/core';
+import { ICON_TYPE } from '@spartacus/storefront';
+import { MessageEvent } from '../messaging';
+
+export const FROM_CUSTOMER = 'C';
```

Comment:
we getting rid of this later?

Reply:
nope, it's used for showing in Avatar when there's no author for message found.

Reply:
maybe in the future we will use initials like they do in other apps, this is fine for first release

Reply:
We are using initials, this is when there is no user name provided. since it is shared component, we decided to keep something like this.